frameworks/digital-outcomes-and-specialists/manifests/edit_brief.yml

```diff
@@ -57,128 +57,102 @@
     - summary
 
 -
-  name: Shortlist criteria
+  name: Shortlist and evaluation process
   editable: True
   step: 2
   description: |
-    After you’ve published your requirements, suppliers will apply and tell you:
+    There are 3 stages to finding the supplier that best meets your needs.
 
-    - when they can start work
-    - if they have the essential skills and experience
-    - which nice-to-have skills and experience they have
+    **1\. View suppliers who applied**<br />
```

Comment:
We shouldn't embed HTML in the content.  These descriptions are passed through a Markdown filter (which is why the `**` works for emboldening things).  To get a line break to render I *think* it just needs another line break in the content here, e.g. 
```
**1\. View suppliers who applied**

When suppliers apply...
``` 

Although that will probably give `<p>` in the rendered page, which might be what you were trying to avoid here.

But we really really shouldn't start embedding HTML in the content.